### PR TITLE
chore: read all messages from a file

### PR DESCRIPTION
**Pull request by BeanieMen**

### CODE REVIEW

File: test/should-not-reply.spec.ts [hunk 1]
```diff
@@ -0,0 +1,12 @@
+import { client } from "./fixtures";
+import { readFileLines } from "./readFile";
+console.log(client);
```
Comment (nopeless) [inline test/should-not-reply.spec.ts]: remove

File: test/should-not-reply.spec.ts [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+const messages: string[] = readFileLines("./test/test-messages.txt")
+describe(`should not reply`, () => {
+	for (const message of messages) {
+		test(message, async () => {
+			const m = await client.send(message);
+			expect(m.mockReplies).to.have.length(0);
```
Comment (nopeless) [inline test/should-not-reply.spec.ts]: should be
`expect(..., Client shouldl not reply to '${message}'`

File: test/should-not-reply.spec.ts [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { client } from "./fixtures";
+import { readFileLines } from "./readFile";
+console.log(client);
+const messages: string[] = readFileLines("./test/test-messages.txt")
+describe(`should not reply`, () => {
+	for (const message of messages) {
+		test(message, async () => {
```
Comment (nopeless) [inline test/should-not-reply.spec.ts]:
```js
test('file', async () => {
  for (const message of messages) { ... }
  }
)
```

Current test scheme spams console too much

**Review by nopeless — CHANGES_REQUESTED** [review]

**Comment by BeanieMen** [comment]
Hey nop, i forgor what the issue wanted me to do, i am gonna pick this up again. Can you explain what i had to do

**Comment by BeanieMen** [comment]
HOW DID I WRITE SUCH SHIT CODE WTF

**Comment by BeanieMen** [comment]
Please?

**Comment by nopeless** [comment]
fix eslint errors

the goal is to check if the bot properly ignores normal messages

**Comment by BeanieMen** [comment]
> fix eslint errors
> 
> the goal is to check if the bot properly ignores normal messages

idk what i did in this pr
I am getting no eslint errors and using your tests i can confirm it is ignoring normal messages

**Comment by BeanieMen** [comment]
It is failing one test called "tl shorthand" tho

**Comment by nopeless** [comment]
@BeanieMen read PR title

**Pull request closed without merging**